app/views/pages/homepage/active_homepage.html.erb

```diff
@@ -47,83 +62,101 @@
           <span>You can sign up anytime between October 1 and October 31.</span>
         </li>
       </ul>
-      <h2 class="title is-2">Rules</h2>
+
+      <h2 class="title is-2" id="rules">
+        <a href="#rules">Rules</a>
+      </h2>
       <p>
-       To earn your Hacktoberfest tee or tree reward, you must register and make four valid pull requests (PRs) between October 1-31 (in any time zone). PRs can be made to any public repo on GitHub, not only the ones with issues labeled Hacktoberfest. If a maintainer reports your pull request as spam or behavior not in line with the project’s code of conduct, you will be ineligible to participate. This year, the first 75,000 participants who successfully complete the challenge will be eligible to receive a prize.
+        To earn your Hacktoberfest tee or tree reward, you must register and make four valid pull requests (PRs) between
+        October 1-31 (in any time zone). PRs can be made to any public repo on GitHub, not only the ones with issues
+        labeled Hacktoberfest. If a maintainer reports your pull request as spam or behavior not in line with the
+        project’s code of conduct, you will be ineligible to participate. This year, the first 75,000 participants who
+        successfully complete the challenge will be eligible to receive a prize.
       </p>
-      <p>Read the <%= link_to 'participation details', details_path, :class => "participation-link" %> to learn how to earn your Hacktoberfest tee or tree reward.</p>
+      <p>Read the <%= link_to 'participation details', details_path, :class => "participation-link" %> to learn how to
+        earn your Hacktoberfest tee or tree reward.</p>
     </div>
   </section>
-  
+
   <section class="home-projects">
     <div class="container">
-      <a name="projects"></a>
-      <h2 class="title is-2">Hacktoberfest projects</h2>
+      <h2 class="title is-2" id="projects">
+        <a href="#projects">Hacktoberfest projects</a>
+      </h2>
       <p>
         You can contribute to any project on GitHub. Here are a few looking for some help:
       </p>
-      <div class="level language-filter">
-        <div class="right">
-          <div class="field">
-            <div class="control">
-              <div class="select">
-                <%= collection_select(:language, :id, Language.all.sort_by(&:name), :id, :name, prompt: "Select language", html_options: {class: 'language-dropdown'}) %>
+      <% if @projects.empty? %>
+        <br/>
+        <p><i>Sorry, we couldn't load any projects to highlight at this time.</i></p>
```

Comment:
I wonder if we could change this to something more proactive? Like "Don't see any projects yet? Browse more projects below"

Reply:
👍 Tweaked the copy to be a bit more CTA-y